### PR TITLE
1657394: 23rd batch of 20 files with new SEO descriptions.

### PR DESCRIPTION
**Pull request by DennisLee-DennisLee**
## Summary

For [User Story 1657394: SEO OKR: [ms.prod(dotnet-framework)] Missing description issue (513)](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1657394).

### CODE REVIEW

File: docs/framework/whats-new/index.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "What's new in the .NET Framework"
```
Comment (gewarren) [inline docs/framework/whats-new/index.md:2]:
```suggestion
title: "What's new in .NET Framework"
```

File: docs/framework/whats-new/index.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
+description: See what's new in various versions of the .NET Framework. Read a summary of key new features and improvements in each version.
```
Comment (gewarren) [inline docs/framework/whats-new/index.md:3]:
```suggestion
description: See what's new in various versions of .NET Framework. Read a summary of key new features and improvements in each version.
```

File: docs/framework/whats-new/obsolete-members.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Obsolete members - .NET Framework"
+description: See the list of type members that are obsolete in .NET 4.5 and later versions. Type members are grouped by assembly. It doesn't list members of obsolete types.
```
Comment (gewarren) [inline docs/framework/whats-new/obsolete-members.md:3]:
```suggestion
description: See the list of type members that are obsolete in .NET 4.5 and later versions. Type members are grouped by assembly. Members of obsolete types are excluded here.
```

File: docs/framework/whats-new/obsolete-types.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Obsolete Types in the .NET Framework"
```
Comment (gewarren) [inline docs/framework/whats-new/obsolete-types.md:2]:
```suggestion
title: "Obsolete Types in .NET Framework"
```

File: docs/framework/whats-new/obsolete-types.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
...
+description: See the list of types that are obsolete in .NET 4.5 and .NET 4.6, organized by assembly. Recommended alternatives are also listed.
```
Comment (gewarren) [inline docs/framework/whats-new/obsolete-types.md:3]:
```suggestion
description: See the list of types that are obsolete in .NET Framework 4.5 and .NET 4.6, organized by assembly. Recommended alternatives are also listed.
```

File: docs/framework/whats-new/whats-new-in-accessibility.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "What's new in accessibility in the .NET Framework"
+description: See what's new in .NET accessibility, starting with .NET 4.7.1. Accessibility features let an app provide the right experience for Assistive Technology users.
```
Comment (gewarren) [inline docs/framework/whats-new/whats-new-in-accessibility.md:3]:
```suggestion
description: See what's new in .NET accessibility, starting with .NET Framework 4.7.1. Accessibility features let an app provide the right experience for Assistive Technology users.
```

**Review by gewarren — APPROVED** [review]
I only reviewed the framework/whats-new articles. I'll leave the framework/ui-automation articles for Andy.

**Review by adegeo — APPROVED** [review]

**Comment by DennisLee-DennisLee** [comment]
#sign-off

**Pull request merged**